domains/tat2008.json

```diff
@@ -0,0 +1,12 @@
+
+    {
+      "owner": {
+        "username": "tat2008",
+        "email": "tuan.trananh2008@gmail.com"
+      },
+
+      "record": {
+        "CNAME": "45e06e35-1c31-4365-bbf3-2b82931a0e33.id.repl.co"
```

Comment:
Internal error?

Reply:
maybe the replit server in your region is error

Reply:
now i just want this domain to redirect to my website, can you check again pls?